libs/logging/src/log_event_ids.ts

```diff
@@ -291,6 +299,56 @@ const SmartcardProgrammedOverrideWriteProtection: LogDetails = {
     'Smartcard is programmed to override a flag protecting writes on the card. By default admin cards can not be written unless write protection is first overridden.',
 };
 
+const TogglingTestMode: LogDetails = {
+  eventId: LogEventId.TogglingTestMode,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User has initiated toggling between test mode and live mode in the current application.',
+};
+
+const ToggledTestMode: LogDetails = {
+  eventId: LogEventId.ToggledTestMode,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User has finished toggling between live mode and test mode in the given application. Success or failure is indicated by the disposition.',
+};
+
+const ClearingBallotData: LogDetails = {
+  eventId: LogEventId.ClearingBallotData,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User has initiated clearing ballot data in the current application.',
+  defaultMessage: 'User is clearing ballot data...',
+};
+
+const ClearedBallotData: LogDetails = {
+  eventId: LogEventId.ClearedBallotData,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User has finished clearing ballot data in the given application. Success or failure is indicated by the disposition.',
+};
+
+const OverridingMarkThresholds: LogDetails = {
+  eventId: LogEventId.OverridingMarkThresholds,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User has initiated overriding the thresholds of when to count marks seen by the scanning module. New mark thresholds specified in the keys `marginal` `definite` and, if defined, `writeInText`.',
+  defaultMessage: 'User is overriding mark thresholds...',
+};
+
+const OverrodeMarkThresholds: LogDetails = {
+  eventId: LogEventId.OverrodeMarkThresholds,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User has finished overriding the thresholds of when to count marks seen by the scanning module. Success or failure is indicated by the disposition. New mark thresholds specified in the keys `marginal` `definite` and, if defined, `writeInText`.',
+};
+const DownloadedScanImageBackup: LogDetails = {
+  eventId: LogEventId.DownloadedScanImageBackup,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User downloaded a backup file of the scanned ballot image files and Cvrs. Success or failure indicated by disposition.',
```

Comment:
Consistently use `CVR`?

```suggestion
    'User downloaded a backup file of the scanned ballot image files and CVRs. Success or failure indicated by disposition.',
```